ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -95,41 +126,58 @@ function AlertHeader({ selectedAlert }: { selectedAlert: Alert }) {
           marginTop={3}
           marginBottom={4}
         >
-          {selectedAlert.reason ?? t('alert')}
+          {customAlertTitle ?? selectedAlert.reason ?? t('alert')}
         </Text>
       </Box>
     </>
   );
 }
 
-function AlertDetails({ selectedAlert }: { selectedAlert: Alert }) {
+function AlertDetails({
+  selectedAlert,
+  customAlertDetails,
+}: {
+  selectedAlert: Alert;
+  customAlertDetails?: React.ReactNode;
+}) {
   const t = useI18nContext();
   const severityStyle = getSeverityStyle(selectedAlert.severity);
   return (
-    <Box
-      key={selectedAlert.key}
-      display={Display.InlineBlock}
-      padding={2}
-      width={BlockSize.Full}
-      backgroundColor={severityStyle.background}
-      gap={2}
-      borderRadius={BorderRadius.SM}
-    >
-      <Text variant={TextVariant.bodySm}>{selectedAlert.message}</Text>
-      {selectedAlert.alertDetails?.length ? (
-        <Text variant={TextVariant.bodySmBold} marginTop={1}>
-          {t('alertModalDetails')}
-        </Text>
-      ) : null}
-
-      <Box as="ul" className={'alert-modal__alert-details'} paddingLeft={6}>
-        {selectedAlert.alertDetails?.map((detail, index) => (
-          <Box as="li" key={`${selectedAlert.key}-detail-${index}`}>
-            <Text variant={TextVariant.bodySm}>{detail}</Text>
-          </Box>
-        ))}
+    <>
```

Comment:
no need to wrap with Fragment <>

Reply:
Thanks for it, removed here.

Reply:
I'm confused since I still see it in the diff here 😅

Reply:
I'm confused since it is still showing 😅  This comment also applies to AlertHeader above. Tested locally to check that this works without the fragments

Reply:
Sorry I pasted the wrong link, I pushed this change to the [PR](https://github.com/MetaMask/metamask-extension/pull/24469/files#diff-70811944eadeada81c400eb541602d61755f131b94a3e21b8f4a833e4b27e994R203) where I was applying the changes requested by Sayalee.